extensions/Khronos/KHR_mesh_compression/README.md

```diff
@@ -0,0 +1,172 @@
+# KHR_draco_geometry_compression
```

Comment:
In glTF, point clouds are represented via `mesh` object by using different `mesh.primitive.mode` value. So they're meshes in glTF terms. I think that extension's name should reflect its formal relations with core spec.

CC @pjcozzi